tests/test-channel-inversion/environment.yaml

```diff
@@ -0,0 +1,7 @@
+channels:
+  - rapidsai
+  - nvidia
```

Comment:
anything special about rapidsai and nvidia here? or could these be any channels. might be kind to future maintenance to leave a note about why you picked these two channels